ts-generator.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "generator": "typechain",
+    "files": "./build/**/*/!(Migrations.json)",
+    "target": "web3-1.0.0",
```

Comment:
This means while generating interacts web3-1.0.0 version will be used. Correct ❓ 
Why package.json web3 version is different from this version 🤔 ?

Reply:
It means this will support `1.0.X` version of web3. This is not tightly related with exact web3 version. It's more about the major version.